packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md

```diff
@@ -60,58 +64,59 @@ You can divide the features of an IAM system into three areas: user experience,
 
 #### Infrastructure
 
-- **Scalability:** Design your solution to allow for user growth, changing use patterns, and evolutionary changes to your applications and infrastructure, without requiring a redesign.
+- **Scalability:** Design your solution to allow for user growth and changing use patterns of your apps and infrastructure, without requiring a redesign.
 
-- **Easy integration with app stack:** Decouple your IAM solution from your applications to allow them to evolve independently and enable multiple apps to use your solution. Enable integration with your apps that optimizes performance by designing an event-based interface that is accessed using an API. Provide an SDK to make integration easier.
+- **Easy integration with app stack:** Maintain your IAM solution separately from your apps. Your apps can evolve independently and you can enable multiple apps to use your IAM solution. Enable integration with your apps that optimizes performance by designing an event-based interface that is accessed using an API. Provide an SDK to make integration easier.
 
-- **Traffic surge protection:** Provide a way to prevent sudden bursts of requests from disrupting usage or operations.
+- **Traffic surge protection:** Prevent sudden bursts of requests from disrupting usage or operations.
 
-- **High availability:** Ensure that all elements of your IAM solution are fully operational whenever your apps are expected to be operable.
+- **High availability:** Ensure that your IAM solution is fully operational when your apps go live.
 
-- **High reliability** (highly redundant): Ensure that all elements of your IAM solution are highly reliable and dependable, and that every element of your solution works correctly, every time.
+- **High reliability:** Ensure that your IAM solution is dependable and that every element of your solution works correctly, every time.
 
 #### Security
 
 - **User storage and password management:** Store and manage information about your users, including their passwords, for authenticating and authorizing users.
 
-- **MFA (Multifactor Authentication):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your applications.
+- **Multifactor Authentication (MFA):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, or biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your apps.
```

Comment:
Is there a better way that we can present this information in this sentence without using "such as" so much?
```suggestion
- **Multifactor Authentication (MFA):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, or biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your apps.
```